apps/prairielearn/src/ee/lib/lti13.ts

```diff
@@ -53,8 +53,8 @@ export const LineitemSchema = z.object({
   resourceId: z.string().optional(),
   resourceLinkId: z.string().optional(),
   tag: z.string().optional(),
-  startDateTime: DateFromISOString.optional(),
-  endDateTime: DateFromISOString.optional(),
+  startDateTime: z.string().datetime({ offset: true }).optional(),
+  endDateTime: z.string().datetime({ offset: true }).optional(),
```

Comment:
I started playing with how Canvas shows these days and realized I had a type wrong.

Reply:
Say more about why this was wrong? `DateFromISOString` uses the `Date` constructor under the hood, so it should be more than capable of handling timezone offsets on its own.

Reply:
I'm fine to revert it and address in another PR. I think it was something zod was verifying (maybe after creating a new assignment on Canvas?) but I didn't save the error message.

Reply:
Yeah, I want you and I to have an understanding of _why_ this change is needed. Please save error messages when you come across them!

Reply:
I started looking into it, but I decided to revert and address somewhere else. I don't want to hold this back.

Reply:
I think that's the right move for now.